README.md

```diff
@@ -87,13 +87,18 @@ polymer-bundler as a library has two exported function.
 
 `polymer-bundler` constructor takes an object of options similar to the command line options:
 
-- `excludes`: URLs to exclude from inlining. URLs may represent files or folders. HTML tags referencing excluded URLs are preserved.
-- `inlineCss`: Inline external stylesheets.
-- `inlineScripts`: Inline external scripts.
-- `sourcemaps`: Honor (or create) sourcemaps for inline scripts
-- `stripComments`: Remove non-license HTML comments.
+- `analyzer`: An instance of `Analyzer` (from `polymer-analyzer` package) which provides analysis of and access to files to operate on.
+- `excludes`: Array of URLs to exclude from inlining. URLs may represent files or folders. HTML tags referencing excluded URLs are preserved.
```

Comment:
Are these urls or file paths? For a url without an extension `src/some/file` do we assume it is a directory?